lib/t/matches/match.ex

```diff
@@ -8,8 +8,6 @@ defmodule T.Matches.Match do
     field :user_id_1, Ecto.Bigflake.UUID
     field :user_id_2, Ecto.Bigflake.UUID
 
-    field :exchanged_voicemail, :boolean
```

Comment:
this seems like a breaking change, would old client be ok with this field removed from responses?

Reply:
with this in particular - yes (https://github.com/getsince/test5/pull/202/files#diff-d5a373f3ff78460009f5de32b1aa475744f13f1125990d4de348896dc7c3bc42L36);

with one other - no (https://github.com/getsince/test5/pull/202/files#diff-ff2ea48ff2f5f39ca8c03130d73e2dcb1e534ad449e4020fa77ec29657958c17L189)
-> I will revert this one so there is no breaking change